modules/sequence/src/main/scala/lucuma/odb/sequence/flamingos2/longslit/Config.scala

```diff
@@ -98,18 +97,4 @@ object Config:
       DefaultFlamingos2ReadoutMode,
       explicitReadoutMode,
       explicitSpatialOffsets
-    )
-
-  def reconcile(a: Config, modes: List[ObservingMode]): Option[Config] =
```

Comment:
Before, it was possible to compute distinct default binning values across targets in an asterism.  The binning calculation handles this now by selecting the smallest binning across all targets if there are multiple options.